src/components/OverviewCard.jsx

```diff
@@ -42,28 +159,124 @@ const OverviewCard = ({
+              }}
+            className={`${contentStyles.componentIcon}`}
           >
-            <i id={`settingsIcon-${itemIndex}`} className="bx bx-cog" />
+            <i id={`settingsIcon-${itemIndex}`} className="bx bx-dots-vertical-rounded" />
           </button>
+          {canShowDropdown &&
+            <div className={`position-absolute ${elementStyles.dropdown}`} ref={dropdownRef} tabIndex={2} onBlur={()=>setCanShowDropdown(false)}>
+              <MenuItem handler={(e) => {dropdownRef.current.blur(); settingsToggle(e)}} id={`settings-${itemIndex}`}>
+                <i id={`settingsIcon-${itemIndex}`} className="bx bx-sm bx-edit"/>
+                Edit details
+              </MenuItem>
+              <MenuItem handler={toggleDropdownModals}>
+                <i className="bx bx-sm bx-folder"/>
+                Move to
+                <i className="bx bx-sm bx-chevron-right ml-auto"/>
+              </MenuItem>
+              <MenuItem handler={() => setCanShowDeleteWarningModal(true)}>
+                <i className="bx bx-sm bx-trash text-danger"/>
+                Delete item
+              </MenuItem>
+          </div>}
+          {canShowFileMoveDropdown &&
+            <div className={`position-absolute ${elementStyles.dropdown}`} ref={fileMoveDropdownRef} tabIndex={1} onBlur={handleBlur}>
+              <MenuItem className={`d-flex`} handler={toggleDropdownModals}>
+                <i className="bx bx-sm bx-arrow-back"/>
+                Move to
+              </MenuItem>
+              <hr/>
+              {category && !isResource &&
+                <MenuItem handler={() => moveFile('')}>
+                  Unlinked Page
+                </MenuItem>
+              }
+              {allCategories
+                ?
+                allCategories.map((categoryName) => {
+                  if (categoryName !== category) {
+                    return (
+                      <MenuItem key={categoryName} handler={() => moveFile(categoryName)}>
+                        {categoryName}
+                      </MenuItem>
+                    )
+                  }
+                  return null
+                })
+                :
+                  `Loading Categories...`
+              }
+              <hr/>
+              <div className={`d-flex text-nowrap ${elementStyles.dropdownItem}`} onClick={(e)=>{e.preventDefault();e.stopPropagation()}}>
+                <i className="bx bx-sm bx-folder-plus" />
+                <input
+                  type="text"
+                  placeholder={'Create new category'}
+                  value={newCategory}
+                  id={'categoryName'}
+                  className={errorMessage ? `${elementStyles.error}` : null}
+                  onChange={changeHandler}
+                  ref={categoryInputRef}
+                />
+                <LoadingButton
+                  label="Submit"
+                  disabled={errorMessage || !newCategory}
```

Comment:
React interprets the `errorMessage` as a string and throws the following error:

```
Warning: Failed prop type: Invalid prop `disabled` of type `string` supplied to `LoadingButton`, expected `boolean`.
    in LoadingButton (at OverviewCard.jsx:253)
    in OverviewCard (at CollectionPagesSection.jsx:136)
    in div (at CollectionPagesSection.jsx:124)
    in div (at CollectionPagesSection.jsx:120)
    in CollectionPagesSection (at CategoryPages.jsx:80)
    in div (at CategoryPages.jsx:74)
    in div (at CategoryPages.jsx:71)
    in CategoryPages (at ProtectedRoute.jsx:41)
    in Route (at ProtectedRoute.jsx:12)
    in ProtectedRoute (at App.jsx:95)
    in ProtectedRouteWithProps (at App.jsx:113)
    in Switch (at App.jsx:109)
    in div (at App.jsx:100)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App.jsx:99)
    in App (at src/index.jsx:7)
```

So we should change it to `!!errorMessage`

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/219/commits/c0548f41c426f645713a708f89d5d142c454581b